multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -81,3 +81,8 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 
 # Hyperkube image repository to use.
 # hyperkubeImageRepo: quay.io/coreos/hyperkube
+
+# AWS Tags for cloudformation stack (formatted as a list of key-value pairs)
+#tags:
+#- Name: "Kubernetes" 
+#- Environment: "Production"
```

Comment:
I decided to go with the list so that an arbitrary amount of tags could be specified. 

Key value pairs because that's what the Tag constructor accepts.
https://godoc.org/github.com/aws/aws-sdk-go/service/cloudformation#Tag